app/views/mortgage_calculator/property_tax_calculator/show.html.erb

```diff
@@ -14,11 +14,15 @@
         <h1 class="stamp-duty__heading"><%= I18n.t("#{i18n_locale_namespace}.heading") %></h1>
         <h2 class="intro stamp-duty__subheading"><%= I18n.t("#{i18n_locale_namespace}.title") %></h2>
         <p>
-          <%= I18n.t(
-            "#{i18n_locale_namespace}.subtitle",
-            amount: second_home_threshold,
-            additional_property_rate: additional_property_rate
-          ) %>
+          <%= I18n.t("#{i18n_locale_namespace}.subtitle") %>
+          <a href="https://www.moneyadviceservice.org.uk/en/tools/house-buying/land-and-buildings-transaction-tax-calculator-scotland" target="_blank">
```

Comment:
This URL (and the subsequent one) should be different for English and Welsh language versions, e.g. `/en/tools/house-buying/land-and-buildings-transaction-tax-calculator-scotland` in English and `/cy/tools/prynu-ty/cyfrifiannell-treth-trafodion-tir-ac-adeiladau-alban` in Welsh. For this we should be loading them from the translation file.

Reply:
Fixed.

Reply:
👍